internal/app/singularity/remote_status.go

```diff
@@ -127,6 +140,48 @@ func doStatusCheck(name string, sp endpoint.Service) *status {
 	return &status{name: name, uri: uri, status: "OK", version: version}
 }
 
+func printLoggedInIdentity(config *scslibclient.Config) error {
+	client := http.Client{Timeout: 5 * time.Second}
+	path := UserServicePath
+	endPoint := config.BaseURL + path
+
+	req, err := http.NewRequest(http.MethodGet, endPoint, nil)
+	if err != nil {
+		return fmt.Errorf("error creating new request: %v", err)
+	}
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", config.AuthToken))
+	res, err := client.Do(req)
```

Comment:
Could we re-use `config.HTTPClient` here instead of creating a new `http.Client`?

```suggestion
	res, err := config.HTTPClient.Do(req)
```